partner_store/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
This file is supposed to e generated from documents in the readme/ subdirectory but this directory is missing. 

Additionally the content of the file is suspicious. I see no "geospatial support" in the module, for instance.

Reply:
Better like this